documentation/en/api-scripting-support.md

```diff
@@ -1,25 +0,0 @@
-# Remote API Support
-
-You may want to delegate the work **Lotus Miner** or **Lotus Node** performs to other machines. 
-Here is how to setup the necessary authorization and environment variables.
-
-## Environment variables
-
-Environmental variables are variables that are defined for the current shell and are inherited by any child shells or processes. Environmental variables are used to pass information into processes that are spawned from the shell.
-
-Using the [JWT you generated](https://lotu.sh/en+api#how-do-i-generate-a-token-18865), you can assign it and the **multiaddr** to the appropriate environment variable.
-
-```sh
-# Lotus Node
-FULLNODE_API_INFO="JWT_TOKEN:/ip4/127.0.0.1/tcp/1234/http"
-
-# Lotus Miner
-MINER_API_INFO="JWT_TOKEN:/ip4/127.0.0.1/tcp/2345/http"
-```
-
-You can also use `lotus auth api-info --perm admin` to quickly create _API_INFO env vars
```

Comment:
I don't see references to this command in the new docs

Reply:
It's here for miners https://docs.filecoin.io/mine/lotus/seal-workers/#create-an-authentication-token, in the context where it's useful.

This for the rest: https://docs.filecoin.io/build/lotus/api-token-generation/

I'll add it also here https://docs.filecoin.io/get-started/lotus/configuration-and-advanced-usage/#controlling-a-remote-daemon

Reply:
ok, I remember now. That creates API info vars that are only correct if the user has provided the external IP for Lotus in the config (I guess). Otherwise it is going to show a localhost address. This is a possible place for pitfalls. If they have to manually write the multiaddress because they just got a token with `create-token`, users will have to pay a bit more attention.

Reply:
Does it use `RemoteListenAddress` at all? It seems to pick up `~/.lotus/api`. That is going to be ultimately set to 127.0.0.1. It just does not seem like a very useful command.

Reply:
RemoteListenAddress is used for cross-worker sector transfers.

~/.lotus/api should be updated after changing the api listen address in the config

Reply:
Yes but only the port. The output of the command is never going to give the actual IP:port multiaddress for a remote node, only for a local one.

Reply:
https://github.com/filecoin-project/filecoin-docs/pull/435